.github/workflows/coverage_reporting.yml

```
@@ -43,13 +43,29 @@ jobs:
       - name: Run receptor tests with coverage
         run: make coverage
 
-      - name: Upload coverage reports to Codecov
-        uses: codecov/codecov-action@v5
-        with:
-          fail_ci_if_error: true
-          verbose: true
+      - name: SonarCube Static Scans (on push)
+        uses: SonarSource/sonarqube-scan-action@v6
+        if: github.event_name == 'push' && startsWith(github.repository, 'ansible') && endsWith(github.repository, '/receptor')
```

```suggestion
        if: github.event_name == 'push' && github.repository == 'ansible/receptor'
```
Do you think this would work? Is the use of startsWith/endsWith a workaround for something?

---

I like this a lot better. Thanks!